py/packages/genkit/src/genkit/core/action.py

```diff
@@ -137,7 +137,7 @@ def __init__(
             on_chunk: The callback to invoke when a chunk is received.
             context: The context to pass to the action.
         """
-        self.__on_chunk = (
+        self.on_chunk = (
```

Comment:
instead of making on_chunk public, let's add a new field `is_streaming`?

Reply:
> instead of making `on_chunk` public, let's add a new field `is_streaming`?

good idea, done